src/ViewModels/src/RemoteDebug/RemoteDebugViewModel.cs

```diff
@@ -265,13 +268,20 @@ public async Task BeginRemoteDebuggingAsync(string appName)
                 Close();
                 return;
             }
+            
+            LoadingMessage = $"Checking for debugging agent on {AppToDebug.AppName}...";
 
             _debugAgentInstalled = await CheckForVsdbg(AppToDebug.Stack);
             if (!_debugAgentInstalled)
             {
-                Option1Text = $"Push new version of \"{AppToDebug.AppName}\" to debug (project \"{_projectName}\")";
-                PromptAppResolution($"Unable to locate debugging agent on \"{AppToDebug.AppName}\".");
-                return;
+                var vsVersion = "latest"; // TODO: make this dynamic
+                LoadingMessage = $"Installing debugging agent for {AppToDebug.AppName}...";
+                var installationResult = await _vsdbgInstaller.InstallVsdbgForCFAppAsync(AppToDebug, vsVersion);
```

Comment:
Yeah makes sense -- if & when this var changes from hardcoded to dynamic, the `VsdbgInstaller` could do that lookup anyway (since I can't think of a time when you'd want to call this method and have the debugging agent get installed for a version of VS that isn't the one you're using...)